collection-scripts/gather_nodes

```diff
@@ -47,6 +47,26 @@ do
         fi
     done
 
+    oc exec -it $pod -n node-gather -- [ -d /host/opt/cni/bin ]
+    if [[ $? -eq 0 ]]; then
+        CNI_BIN_PATH="${NODES_PATH}/$node/opt/cni/bin"
+        mkdir -p ${CNI_BIN_PATH}
```

Comment:
you reuse `$NODE_PATH` which is defined on line 33 instead of `${NODES_PATH}/$node`

Reply:
Done